projects/adrv9001/src/app/headless.c

```diff
@@ -81,47 +89,44 @@ int get_sampling_frequency(struct axi_adc *dev, uint32_t chan,
 static int32_t iio_run(struct iio_axi_adc_init_param *adc_pars,
 		       struct iio_axi_dac_init_param *dac_pars)
 {
-	char *names[] = {
-		"axi_adc1",
-		"axi_dac1",
-		"axi_adc2",
-		"axi_dac2"
-	};
 	struct iio_axi_adc_desc *adcs[IIO_DEV_COUNT];
 	struct iio_axi_dac_desc *dacs[IIO_DEV_COUNT];
 	struct iio_data_buffer iio_dac_buffers[IIO_DEV_COUNT];
 	struct iio_data_buffer iio_adc_buffers[IIO_DEV_COUNT];
 	struct iio_device *iio_descs[IIO_DEV_COUNT * 2];
 	struct iio_app_device app_devices[IIO_DEV_COUNT * 2] = {0};
-	int32_t i, ret;
+	int32_t d, ret;
+	int32_t a; // linear iterator for iio_descs and app_devices flat arrays
 
-	for (i = 0; i < IIO_DEV_COUNT; i++) {
+	for (d = 0; d < IIO_DEV_COUNT; d++) {
```

Comment:
why replacing `i` with `d`? If it has a specific meaning, I'd add a comment similar to the one for `a`

Reply:
i reverted that rename - it was indeed not needed